homeassistant/components/linear_garage_door/__init__.py

```diff
@@ -0,0 +1,32 @@
+"""The Linear Garage Door integration."""
+from __future__ import annotations
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import Platform
+from homeassistant.core import HomeAssistant
+
+from .const import DOMAIN
+from .coordinator import LinearUpdateCoordinator
+
+PLATFORMS: list[Platform] = [Platform.COVER]
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+    """Set up Linear Garage Door from a config entry."""
+
+    coordinator = LinearUpdateCoordinator(hass, entry)
+
+    await coordinator.async_config_entry_first_refresh()
+
+    hass.data.setdefault(DOMAIN, {})[entry.entry_id] = {"coordinator": coordinator}
```

Comment:
Just set it equal to the coordinator. A one item dict is unneeded. If you plan to have multiple pieces of data in hass.data for your domain, create a model.